pandas/io/formats/style.py

```diff
@@ -47,6 +49,12 @@ def _mpl(func):
         raise ImportError(no_mpl_message.format(func.__name__))
 
 
+_ApplyArgs = Tuple[
+    Callable[[FrameOrSeries], FrameOrSeries], Axis, Optional[_IndexSlice]
+]
+_Kwargs = Dict[str, Any]
```

Comment:
The Kwargs alias is not for annotating kwargs in function signiture. The kwargs dict is stored in a local variable. I appreciate this could be confusing.

It was to make `self._todo = []  # type: List[Tuple[Callable, _ApplyArgs, _Kwargs]]` more readable.

i'll inline _Kwargs to avoid this confusion.